ci_build/Dockerfile

```diff
@@ -12,8 +12,6 @@ ADD . /tfhs
 WORKDIR /tfhs
 
 RUN \
-    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 575159689BEFB442 && \
```

Comment:
This was slightly more end-to-end secure, but I guess we place ultimate trust into github anyway, so no big deal.

Reply:
Whoops, got trigger happy. Restored it.